Gemfile.lock

```diff
@@ -36,6 +64,7 @@ GEM
     rubocop-ast (1.11.0)
       parser (>= 3.0.1.1)
     ruby-progressbar (1.11.0)
+    ruby2_keywords (0.0.5)
```

Comment:
lots of Gems being used here, what are they all for? Interested to know where they are used in your program and what they do